src/login/logind-dbus.c

```diff
@@ -2720,7 +2720,6 @@ const sd_bus_vtable manager_vtable[] = {
         SD_BUS_PROPERTY("NCurrentInhibitors", "t", property_get_current_inhibitors, 0, 0),
         SD_BUS_PROPERTY("SessionsMax", "t", NULL, offsetof(Manager, sessions_max), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("NCurrentSessions", "t", property_get_current_sessions, 0, 0),
-        SD_BUS_PROPERTY("UserTasksMax", "t", NULL, offsetof(Manager, user_tasks_max), SD_BUS_VTABLE_PROPERTY_CONST),
```

Comment:
hmm, so far we avoided fully removing bus props and methods, but instead kept them and marked them with SD_BUS_VTABLE_HIDDEN, so that they don't appear anymore... In this case it might make sense to harcode the value to "unlimited"...